config/samples/core_v1alpha1_componentplanchartmuseum.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: ComponentPlan
+metadata:
+# labels:
```

Comment:
Option 1: Delete these comments
Option 2: Update to correct comments 

Both are fine.